### PR TITLE
Remove api from readme

### PR DESCRIPTION
**Pull request by koresar**
Plus some more code examples, more ES6, more cross-links to docs, move .md files to /docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,390 +212,4 @@ var myBar = bar({name: 'Moe\'s'});
...
-
-  var t = myThing();
-
-  t.thing; // 'initialized',
-
-  t.foo(); // 'foo',
-
-  t.bar(); // 'bar'
-```
+For more examples see [API](docs/API.md) and [advances examples](docs/advanced_examples.md).
```
Comment (troutowicz) [inline README.md]: For more examples see the [API](docs/API.md) and the [advances examples](docs/advanced_examples.md).

File: docs/API.md [hunk 2]
```diff
@@ -3,41 +3,48 @@
...
+const conn = DbConnection(); // Open a DB connection
+const conn2 = DbConnection({ connection: conn.dbConnection }); // reusing existing
+conn.close(); // Close the conneciton.
 
 // Connect to a different DB.
-var localConn = DbConnection({ connectionConfig: 'mongodb://localhost' });
+const localConn = DbConnection({ connectionConfig: 'mongodb://localhost' });
 ```
 
 **Source: stampit.js**
```
Comment (troutowicz) [inline docs/API.md]: I think this line can be removed.

File: docs/API.md [hunk 3]
```diff
@@ -62,6 +69,23 @@ components that are passed in or composed.
...
+  },
+  refs: {
+    defaultFactor: 1
+  },
+  init() {
+    this.factor = this.factor >= 0 ? this.factor : this.defaultFactor;
+  }
+});
+
+var objectInstance = stamp({factor: 1.1});
```
Comment (troutowicz) [inline docs/API.md]: I see a few variables being declared with `var`. Should do another pass and fix those. :)

File: README.md (hunk 1 above)
Comment (koresar) [inline README.md]: ty

File: docs/API.md (hunk 2 above)
Comment (koresar) [inline docs/API.md]: Good point

File: docs/API.md (hunk 3 above)
Comment (koresar) [inline docs/API.md]: Yep. Forgot to remove this one.

**Comment by ericelliott** [comment]
:+1:

**Comment by koresar** [comment]
Ready to review and merge. Please, check.

File: docs/API.md [hunk 4]
```diff
@@ -312,5 +412,88 @@ Or `.static()` ...
 Or even `.compose()` ...
 
 ```js
-  var obj = abstractStamp.compose(concreteStamp, additionalStamp, utilityStamp).create();
+  const obj = abstractStamp.compose(concreteStamp, additionalStamp, utilityStamp).create();
+```
+
+
+## Chaining methods
```
Comment (troutowicz) [inline docs/API.md]: These examples are still using `var`.

Comment (koresar) [inline docs/API.md]: One thing at a time Tim. I'm not changing text at this point of time. Just moving.

There are much more changes to ES6-ify examples than just var->const. If you want to ES6-ify them all I welcome you to supply PR.

This issue name is **Remove api from readme**. It doesn't say "make all examples ES6".

If no other objections present, I'll merge this in few hours.

Comment (koresar) [inline docs/API.md]: However, some examples are ES5, some are ES6. No good. :-\

Okay. I'll spend some more time on that and ES6-ify all the text.

Comment (koresar) [inline docs/API.md]: Done

Comment (troutowicz) [inline docs/API.md]: > One thing at a time Tim. I'm not changing text at this point of time. Just moving.

You have introduced several new examples in this PR, this is not just a "moving" commit. I'm not asking for anything that you have not already started to do already.

> Done

:+1:

**Pull request merged**